docs/_debt/DEBT-014-friction-residuals.md

```
@@ -0,0 +1,448 @@
+**Proposed feature:**
+```bash
+# Scan for new markets with quick research
+kalshi scan new-markets --hours 24 --categories politics,ai,tech --research
+```
+
+**Design questions:**
+1. How to detect "new" markets (`created_time` field)?
+2. Should this auto-trigger Exa research?
+3. What's the signal-to-noise threshold?
+
+**✅ INVESTIGATED (2026-01-11): Liquidity Filtering Analysis**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)